cl/recap/tasks.py

```diff
@@ -158,10 +160,17 @@ def do_pacer_fetch(fq: PacerFetchQueue):
     result = None
     if fq.request_type == REQUEST_TYPE.DOCKET:
```

Comment:
Hm, the API is different, and the logic here is different. Should we add a new REQUEST_TYPE for `DOCKET_APPELLATE`? I'm not sure, but maybe?